### PR TITLE
HardWareAsAService.md support disclaimer

### LINKED ISSUE #853: Add info to docs about using BTCPay with other services
A lot of comments in the chat come from users with BTCPay on things like MyNode, Umbrel, Raspiblitz, etc. We should mention in docs that all of these are different ways of implementing BTCPay and we are not able to answer questions about those setups. 

Discussed in [this docs meeting](https://github.com/btcpayserver/organization/issues/23). Talked about adding this disclaimer to Hardware As a Service document.

### PR DESCRIPTION
**Pull request by Zaxounette**
closes #853

* Adds a disclaimer in HardwareAsAService.md stating that questions, bugs and issues should be directed to providers.

### CODE REVIEW

**Review by britttttk — COMMENTED** [review]
Maybe we can link to this https://docs.btcpayserver.org/Troubleshooting/#4-asking-for-help document somehow or point them to the troubleshooting doc... 

 It would be great if we could connect the idea that these are essentially "custom deployments" of btcpay and the community does not provide extensive support for these types of deployments (like the warning in the link).

Not sure how to word that... what do you think @Zaxounette

**Comment by Zaxounette** [comment]
I feel like pointing them to `asking for help` is giving them the stick to beat us, instead of the HAAS providers.

As for the warning, I might just as well copy it entirely and add it on top of the other.

**Comment by dennisreimann** [comment]
I agree with @Zaxounette. We should be consistent when we say we don't offer support for external services – pointing to the Troubleshooting page kind of contradicts that. 

Also think that adding such a disclaimer to the Troubleshooting page makes sense. 👍

**Comment by Zaxounette** [comment]
Okay so:

* 0171f06 did a second pass on the HAAS.md disclaimer to refine it a bit.
* 0669502 Added a sentence in troubleshooting.md at the end of the already existing disclaimer indicating explicitly that it includes HAAS products.

File: docs/HardwareAsAService.md [hunk 1]
```diff
@@ -16,8 +16,9 @@ Here you can find a non-exhaustive list of companies that provide such a service
 - [RaspiBlitz](https://shop.fulmo.org/raspiblitz/)
 - [Umbrel](https://getumbrel.com/)
 
-:: Warning
-BTCPay Server cannot answer questions relating to integrations made by third-party entities. Please report bugs and issues directly to the entity that provided you with BTCPay Server inside their product.
-::
+:::danger
```
Comment (pavlenex) [inline docs/HardwareAsAService.md:19]: Please do not use `dangerous` unless something is really dangerous. In this case a blue mark is absolutely fine. If we keep using dangerous often, people may ignore it.

Comment (Zaxounette) [reply docs/HardwareAsAService.md:19]: Good point.

By blue mark, you mean :::tip ?

Comment (pavlenex) [reply docs/HardwareAsAService.md]: Personally, I don't think it's a warning either. It's simply a prominent indicator that officially something isn't supported.

**Review by dennisreimann — COMMENTED** [review]
Pav's note about the type is good, other than that it LGTM.

**Review by britttttk — APPROVED** [review]

**Comment by britttttk** [comment]
Nice one @Zaxounette much better explanation

**Comment by Zaxounette** [comment]
After discussion with Dennis, 6127a81 pushes the disclaimer up above fold for early view by the user.

Also adds a title "please note" to the container.

ACK.

**Pull request merged**